webapi/src/main/scala/org/knora/webapi/messages/v1/responder/resourcemessages/ResourceMessagesV1.scala

```diff
@@ -193,6 +197,28 @@ case class ResourceCreateRequestV1(resourceTypeIri: IRI,
                                    userProfile: UserProfileV1,
                                    apiRequestID: UUID) extends ResourcesResponderRequestV1
 
+case class OneOfMultipleResourceCreateRequestV1(resourceTypeIri: IRI,
```

Comment:
Please add Scaladoc comments to your classes. The syntax is described here:

http://docs.scala-lang.org/overviews/scaladoc/for-library-authors.html